src/gui/transferlistdelegate.cpp

```diff
@@ -69,8 +69,12 @@ void TransferListDelegate::paint(QPainter *painter, const QStyleOptionViewItem &
     case TransferListModel::TR_PROGRESS:
         {
             const int progress = static_cast<int>(index.data(TransferListModel::UnderlyingDataRole).toReal());
+            const bool isEnabled = !index.data(TransferListModel::AdditionalUnderlyingDataRole).toBool();
```

Comment:
You should check the data of status column instead.

Reply:
Done.